safe_eth/eth/clients/etherscan_client.py

```diff
@@ -320,7 +320,7 @@ def __init__(
         self.request_timeout = request_timeout
 
     def build_url(self, path: str):
-        url = urljoin(self.base_api_url, path)
+        url = urljoin(self.base_api_url, f"api?{path}")
```

Comment:
Why api is out of the path?

Reply:
Maybe the problem is the name of the parameter. I updated it as `query` [here](https://github.com/safe-global/safe-eth-py/pull/1407/commits/9b8049300f80f04aa75dd4b1f643d0735ab3d0ed)